plugins/github-release-manager/src/components/ResponseStepList/ResponseStepList.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { PropsWithChildren } from 'react';
+import { List, CircularProgress } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
```

Comment:
stupid query: any reason these can't be merged into line 17 as just imports off the main `@material-ui/core` package? or is that just a matter of style preference?

Reply:
Probably due to auto-imports 🙂 

Fixed in [latest commit](https://github.com/backstage/backstage/pull/5299/commits/c003e06cc004cc085153112af6240a30e7d73200).